download.ics

```diff
@@ -0,0 +1,95 @@
+BEGIN:VCALENDAR
```

Comment:
Moved to the data folder and renamed. This would be useful to have when no lunches are available on the calendar - e.g. during breaks.